engine/src/main/resources/templates/Advisor/deactivatedRecommendationInstantEmailBody.html

```diff
@@ -0,0 +1,140 @@
+            </tr>
+            <!-- end content block -->
+            </tbody>
+        </table>
+    </td>
+</tr>
+<!-- end body section -->
+
+<!-- Body section -->
+<tr>
+    <td class="rh-body">
+        <table class="rh-content rh-m-plain" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td class="rh-content__block">
+                    <table width:="" 100%;="" class="rh-data-table rh-m-bordered">
+                        <thead>
+                        <tr>
+                            <th>Deactivated recommendations</th>
+                            <th>Total Risk</th>
+                            <th>Affected Systems</th>
+                            <th>Deactivation Reason</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {#each action.events}
+                            <tr>
+                                <td>{it.payload.rule_description}</td>
+                                <td class="rh-severity">
+                                    {#switch it.payload.total_risk}
+                                    {#case '1'}
+                                    <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_low.png" alt="Low severity" width="55" border="0">
+                                    {#case '2'}
+                                    <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_moderate.png" alt="Moderate severity" width="85" border="0">
+                                    {#case '3'}
+                                    <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_important.png" alt="Important severity" width="86" border="0">
+                                    {#case '4'}
+                                    <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_critical.png" alt="Critical severity" width="70" border="0">
+                                    {/switch}
+                                </td>
+                                <td>{it.payload.affected_systems}</td>
+                                <td>{it.payload.deactivation_reason}</td>
+                            </tr>
+                        {/each}
+                        </tbody>
+                    </table>
+                </td>
+            </tr>
+            <!-- end content block -->
+            </tbody>
+        </table>
+    </td>
+</tr>
+
+<!-- Body section -->
+<tr>
+    <td class="rh-body">
+        <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td class="rh-content__block">
+                    <p>
+                        Hi {{user.first_name}},
```

Comment:
Yes, the user first name will be available because we retrieve that information from RBAC using the account number we receive in the Kafka message. However, the template needs a small change:
```suggestion
                        Hi {user.firstName},
```